src/System.IO.Compression/src/System/IO/Compression/Deflater.cs

```diff
@@ -251,5 +252,5 @@ private ZErrorCode Deflate(ZFlushCode flushCode)
         }
         #endregion  // helpers & native call wrappers
 
-    }  // internal class DeflaterZLib
+    }  // internal class Deflater
```

Comment:
I seldom see them. I think they're probably just a feature of the style of whoever wrote them. I'll  remove them for consistency since most classes don't have them.